src/test/java/BaseTest.java

```diff
@@ -1,18 +1,36 @@
 import configuration.DriverManager;
+import org.apache.commons.io.FileUtils;
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
+import org.openqa.selenium.OutputType;
+import org.openqa.selenium.TakesScreenshot;
 import org.openqa.selenium.WebDriver;
 
+import java.io.File;
+
+
 public class BaseTest {
     private WebDriver driver;
     private final static String MAIL_URL = "https://mail.yandex.com";
+    public static final String SCREENSHOTS_PATH = System.getProperty("user.dir") + "\\src\\test\\resources\\screenshots\\";
```

Comment:
it's better to use `target` folderб it will be possible to clean the project (maven command 'mvn clean' deletes this folder with unnecessary files). Use this path: `target/screenshots/`